### DISCUSSION TITLE
How do u actually config?

### DISCUSSION THREAD

**Discussion by hungrypotat** [SkidderMC/FDPClient#947] [Q&A]
I was mostly an anarchy player and reasonable new to ghost clients, how would you make good cfg to bypass anticheats. 
Step by step would be nice and any replies are appreciated :)

**Comment by ghost** [comment]
Hello hungrypotat! 
I do not cheat on servers, only for learning purposes, but I can tell you how I and many others would do it:

1. what anticheat has the network? 
1.2 Try commands and see if the answer is different eg:
/verus (Verus Anticheat) "Unknown Command" Not the Anticheat
/ncp (no cheat plus) "You don't have access to the command" Correct anticheat
(Or you google and see if someone wrote about it on Reddit etc.)
1.3 is there more than one anticheat?

2. what modules are there for this anticheat? 
2.2 e.g.: "NCP Bhop" so there are modules for this anticheat 
2.3 Are the modules up to date, so they work for this version of the anticheat?
2.4 Is there a Youtube video, if someone has made a config for this anticheat in the near future?

3. and most important step: try what works and what does not. Vanilla Fly will probably not go through, so let's see if "NCP slime fly" goes. Or you can start with 4 range in the killaura.
3.2 How does the server/anticheat react? Do you get flagged (reset), get kicked or banned? or does it go through and you can play with it? Before that, decide if you want to cheat Rage or just cheat inconspicuously.
3.3 know if you will be banned by mods or if the anticheat has detected you.
3.4 Always start gently with the settings and work your way up until you realize that the server reacts to it and you go back a step again

4. can the client bypass the anticheat and how hard are the anticheat settings set? Anticheats like Intave and Polar.top often have hard settings, where e.g. FDp has no chance, but since the anticheat also likes to flag the legitimate players, many serverowner set these settings down and they are easier to bypass.
4.2 For which anticheats are the clients designed and how adjustable is the client?
I like FDP because it bypasses many anticheats and you can set killaura and scaffold very differently. The best anticheats to bypass are: 
Verus
Vulcan
Grim
NCP (No cheat plus)
Matrix
Kayru
Watchdog
Spartan
and many more

I'm sure there are some more things to consider that I can't think of right now, but you'll learn them over time. Important to note: Have enough accounts, best for cracked server, because you don't need a real account and have a VPN or a router that changes your Ip on reboot. Have fun and a good evening / day!
